src/java.base/share/classes/sun/security/util/DisabledAlgorithmConstraints.java

```diff
@@ -130,21 +130,14 @@ public DisabledAlgorithmConstraints(String propertyName,
         disabledAlgorithms = getAlgorithms(propertyName);
 
         // Check for alias
-        int ecindex = -1, i = 0;
+        Pattern INCLUDE_PATTERN = Pattern.compile("include " + PROPERTY_DISABLED_EC_CURVES, Pattern.CASE_INSENSITIVE);
+        Matcher matcher;
```

Comment:
I think the performance impact should be trivial if moving the "matcher" declaration into the for-loop block.